exercises/exercise_2/README.md

```diff
@@ -1,219 +1,195 @@
+
+    ```bash
+    $ docker app init voting-app --description "Voting App" --maintainer "dapworkshop:dapworkshop@docker.com"
+    Created "voting-app.dockerapp"
+    ```
+
+3. If you run `ls`, you'll see a new directory. The name is your app name with a `.dockerapp` suffix. Let's look inside the directory.
+
+    ```bash
+    $ ls
+    docker-compose.yml  voting-app.dockerapp
+    $ ls voting-app.dockerapp/
+    docker-compose.yml  metadata.yml        parameters.yml
+    ```
+
+4. The compose file is a copy of the file you were working with earlier. If you open the `metadata.yml` file, you'll see the config we specified during initialization.
+
+    ```bash
+    $ cat voting-app.dockerapp/metadata.yml
+    # Version of the application
+    version: 0.1.0
+    # Name of the application
+    name: voting-app
+    # A short description of the application
+    description: Voting App
+    # List of application maintainers with name and email for each
+    maintainers:
+      - name: dapworkshop
+        email: dapworkshop@docker.com
+    ```
+
+
+
+## Merging our Docker Application
+
+While the default app initialization is to create a new folder and create three separate files, you can actually merge everything into a single file!
+
+1. Use the `docker app merge` command to merge the app into a single file. We use the `-o` flag to specify the new file to create.
+
+    ```bash
+    $ docker app merge voting-app -o voting-app-merged.dockerapp
+    ```
+
+2. If you look at the file, you'll see the contents of each of the original files merged together. For brevity, the full contents aren't included below.
+
+    ```bash
+    $ cat voting-app-merged.dockerapp
+    <metadata.yml contents>
+    ---
+    <docker-compose.yml contents>
+    ---
+    <parameters.yml contents>
+    ```
+
+## Splitting the Docker Application
+
+Once merged, the application can be split back into separate files using the `docker app split` command.
+
+1. Run the `docker app split` command to split the file back out into separate files
+
+    ```bash
+    $ docker app split voting-app-merged.dockerapp -o voting-app-split.dockerapp
+    ```
+
+2. Look in the `voting-app-split.dockerapp` directory and you'll see the files split apart.
+
+    ```bash
+    $ ls voting-app-merged.dockerapp
+    docker-compose.yml  metadata.yml        parameters.yml
+    ```
+
+3. At this point, we don't need the split and merged versions anymore. Let's remove them to reduce confusion going forward.
+
+    ```bash
+    $ rm -r voting-app-merged.dockerapp voting-app-split.dockerapp
+    ```
+
+
+## Inspecting our Docker Application
+
+We can use the `docker app inspect` command to get a quick output of all of the services, number of replicas, ports, and the image being used.
```

Comment:
What about adding something like: "An ops can quickly check all the elements before deploying to production, without parsing himself the compose file"?

Reply:
Good idea. Got it.